### PR TITLE
More CSS resources added

### PR DESCRIPTION
**Pull request by Mansiprajapatii**
some more CSS Resources were added.

### CODE REVIEW

**Comment by nataliepina** [comment]
Hi @Mansiprajapatii thank you for contributing. I am merging in pull requests in order and there are some merge conflicts here you'll have to resolve before we can merge. Let me know if you need any help. Thanks!

File: README.md [hunk 1]
```diff
@@ -15,13 +15,16 @@ A collection of the best CSS resources.
 - [Responsive Web design Certification](https://www.freecodecamp.org/learn/2022/responsive-web-design/)
 - [Scrimba](https://scrimba.com/allcourses?price=free&topic=css)
 - [CSS Portal](https://www.cssportal.com/)
+- [Awesome-CSS Learning](https://github.com/micromata/awesome-css-learning)
+- [CSS-Tricks](https://css-tricks.com/)
```
Comment (nataliepina) [inline README.md]: We already have CSS Tricks listed below in the new section, so you can remove this line.

File: README.md [hunk 2]
```diff
@@ -15,13 +15,16 @@ A collection of the best CSS resources.
...
+- [SoloLearn](https://www.sololearn.com/)
```
Comment (nataliepina) [inline README.md]: We might want to link to the exact CSS course https://www.sololearn.com/learn/courses/css-introduction

File: README.md [hunk 3]
```diff
@@ -15,13 +15,16 @@ A collection of the best CSS resources.
 - [Responsive Web design Certification](https://www.freecodecamp.org/learn/2022/responsive-web-design/)
 - [Scrimba](https://scrimba.com/allcourses?price=free&topic=css)
 - [CSS Portal](https://www.cssportal.com/)
+- [Awesome-CSS Learning](https://github.com/micromata/awesome-css-learning)
```
Comment (nataliepina) [inline README.md]: Let's remove this one. It's a bit of a duplication of the work done in this repository.

**Comment by Mansiprajapatii** [comment]
hello @nataliepina

i reforked the repo and commit some other resources, please review them. I
would be happy if they got merged.

**Pull request closed without merging**